src/main/resources/application.properties

```diff
@@ -13,3 +13,13 @@ management.endpoint.health.show-details=never
 
 internal.api.url=${INTERNAL_API_URL}
 private.api.url=${API_URL}
+
+email.producer.appId=accounts-association-api
```

Comment:
use spring.application.name

Reply:
I think the email producer library requires it to be called email.producer.appId